source/containers/Form/Form.tsx

```diff
@@ -232,12 +253,12 @@ const Form: React.FC<Props> = ({
   );
 
   const mainStep = formState.currentPosition.currentMainStepIndex;
-  const [visible, toggleModal] = useModal();
+  const [, toggleModal] = useModal();
   const [scrollViewRef, setRef] = useState<ScrollView>(null);
 
   useEffect(() => {
     if (scrollViewRef && scrollViewRef?.scrollTo) {
-      InteractionManager.runAfterInteractions(() => {
+      void InteractionManager.runAfterInteractions(() => {
```

Comment:
How come we have a `void` keyword here?

Reply:
According to typescript`InteractionManager.runAfterInteractions` returns a promise, hence the `void` keyword